### ISSUE TITLE
Default filter query seems not to work anymore

### ISSUE THREAD

**Issue by B-urb** [B-urb/doclytics#46]
The default filter `NOT tagged=true` until now returned all documents where the tagged field did not exist or was false. This does not seem to work anymore. The paperless logs show: `no documents found for filter`

**Comment by Deathproof76** [comment]
maybe this one https://github.com/B-urb/doclytics/issues/44#issuecomment-2113632546 opening http://paperless-ip:port/api/documents/?query=NOT%20tagged=true works on my end

**Comment by B-urb** [comment]
You are right. That fixes it. I will add that to known issues. Thanks for sharing.
